### ISSUE TITLE
Native Language parts for Store Finder Addresses

### ISSUE THREAD

**Issue by Zeyber** [SAP/spartacus#15371]
Store Finder addresses should have their native languages appended to lang attributes on the containing elements.

![](https://user-images.githubusercontent.com/15215400/110509547-d4aeba00-80c7-11eb-8491-6b18507a0963.png)
_https://github.com/SAP/spartacus/issues/11439#issue-826295021_

We don’t have any information from the backend which language the address will be returned in. We can only assume it will be the local language and return it based on the country. To do that, we’d also need to map country codes to native language codes. (eg. nz => en).

![](https://user-images.githubusercontent.com/11735817/157894933-34e6e427-0f88-488b-908b-3bf2b83845b1.png)
_https://github.com/SAP/spartacus/issues/11439#issuecomment-1065208793_

**Comment by Zeyber** [comment]
@Xymmer What do you think about the approaches suggested here?
- Use the suggested local language mapping based on the country to get the language of the address (can be done using current backend).
- Get language information from backend for address (not currently supported by backend).

**Comment by Zeyber** [comment]
The first suggestion poses issues for countries with addresses in multiple languages (e.g. Canada with English and French). Lets go with the second suggestion and wait until the backend can deliver some language information.
